cmd/ak/cmd/up.go

```diff
@@ -32,11 +32,9 @@ var upCmd = common.StandardCommand(&cobra.Command{
 		}
 
 		<-app.Wait()
+
 		fmt.Println() // End the output with "\n".
 
-		if err := app.Stop(ctx); err != nil {
```

Comment:
Isn't it correct to run `app.Stop` at this point anyway, regardless of system tests?

Reply:
not afaict. there is no reason to run stop after everything stops.